cpg-core/src/main/java/de/fraunhofer/aisec/cpg/graph/ExpressionBuilder.kt

```diff
@@ -329,36 +321,14 @@ fun MetadataProvider.newMemberExpression(
     val node = MemberExpression()
     node.applyMetadata(this, name, rawNode, code, true)
 
-    node.setBase(base)
+    node.base = base
     node.operatorCode = operatorCode
     node.type = memberType
 
     log(node)
     return node
 }
 
-/**
- * Creates a new [StaticCallExpression]. The [MetadataProvider] receiver will be used to fill
- * different meta-data using [Node.applyMetadata]. Calling this extension function outside of Kotlin
- * requires an appropriate [MetadataProvider], such as a [LanguageFrontend] as an additional
- * prepended argument.
- */
-@JvmOverloads
-fun MetadataProvider.newStaticCallExpression(
```

Comment:
I think this means that we should delete the class `StaticCallExpression` and everything is now a `MemberCallExpression`. Do we actually need this subclass of `CallExpression`?

Reply:
The class should have been deleted. I will double-check